tests/test_wsireader.py

```diff
@@ -28,6 +28,8 @@
 has_cucim = has_cucim and hasattr(cucim, "CuImage")
 _, has_osl = optional_import("openslide")
 imsave, has_tiff = optional_import("tifffile", name="imsave")
+_, has_codec = optional_import("imagecodecs")
```

Comment:
Should we add the installation of `imagecodecs` in the CI setup?

Thanks.

Reply:
It's included https://github.com/Project-MONAI/MONAI/blob/f94e76824cf310d908e407394bda3b0507b3d3c0/requirements-dev.txt#L36